packages/squiggle-lang/src/value/simpleValue.ts

```diff
@@ -288,7 +289,12 @@ export function simpleValueFromValue(value: Value): SimpleValue {
           return simpleValueFromAny(value.value);
         case "SampleSetDist": {
           const dist = value.value as SampleSetDist;
-          return [...dist.samples];
+          const fields: [string, SimpleValue][] = [
+            ["vType", "SampleSetDist"],
+            ["samples", [...dist.samples]],
+            ["summary", simpleValueFromAny(dist.summarize())],
+          ];
+          return ImmutableMap(fields);
```

Comment:
Not directly relevant to this PR, but I'm still conflicted about the entire "simple values" concept. I think it tries to serve several purposes, but it's not the best approach for some of these.

So I tried to list all purposes that it serves, here's the list:

1) Serialization for storage and marshaling (where we don't care about readability, only for correctness): new serialization code is strictly better for this, since it handles lambdas and de-duplication correctly.

2) Display value tree in the browser in the condensed way: we can do this with normal `SqValue`s, with alternative widgets or improvements to the current widgets. The benefit is that it could be interactive.

3) Copy/paste values: I believe we could do "rich non-text values" UI if we wanted to, through the browser's native clipboard API. But it's unclear where we want to paste things and if we should allow users to deal with values in this way.

4) LLMs and CLI debugging; value -> string. This is the only valid application, in my impression. Note that for CLI what we want is the colored text, not plain text.

So my impression is that we don't need a parallel "simple values" types hierarchy.

What we need is a highly customizable `value.toString({ ... options })` method. (or maybe something inverted with the visitor pattern, `traverseValue(root, config)`, where you define callbacks for each value type, so that we don't have to keep all this code in `VFoo` class definitions). For everything else, it's easier to deal with normal `Value`s and `SqValue`s.